integration_tests/steps/pv_tests.py

```diff
@@ -38,12 +38,12 @@ def storages_pv_final_selling_rate(context):
             if trade.buyer == storage1.name:
                 # Storage 1 should buy energy offers with rate more than the PV min sell rate
                 assert round(
-                    trade.offer_bid.energy_rate, DEFAULT_PRECISION) >= \
+                    trade.order.energy_rate, DEFAULT_PRECISION) >= \
                     pv.strategy.offer_update.final_rate[market.time_slot]
 
     for market in house2.past_markets:
         assert all(trade.seller == pv.name for trade in market.trades)
-        assert all(round(trade.offer_bid.energy_rate, DEFAULT_PRECISION) >=
+        assert all(round(trade.order.energy_rate, DEFAULT_PRECISION) >=
```

Comment:
```suggestion
        assert all(limit_float_precision(trade.order.energy_rate) >=
```
Good to watch-out in rest of targeted code changes too.

Reply:
@faizan2590 https://github.com/gridsingularity/gsy-framework/pull/296#discussion_r752882210
I can update this one (it's using the default precision and it's a minor change) but if you agree, we can leave these to some time later

Reply:
As you wish but still as this is more like a refactoring PR and not targeting a feature, so still IMO this slight enhancement is worth it